kratos/tests/cpp_tests/utilities/test_math_utils.cpp

```diff
@@ -606,10 +606,10 @@ namespace Kratos
             mat33(2,1) = mat33(1,2);
             mat33(2,2) = 0.972831;
 
-            bool converged = MathUtils<double>::EigenSystem<3>(mat33, vectormat33, eigenmat33);
+            bool converged = MathUtils<double>::GaussSeidelEigenSystem(mat33, vectormat33, eigenmat33);
 
-            BoundedMatrix<double, 3, 3> othermat33 = prod(trans(vectormat33), eigenmat33);
-            BoundedMatrix<double, 3, 3> auxmat33 = prod(othermat33, vectormat33);
+            BoundedMatrix<double, 3, 3> othermat33 = prod(vectormat33, eigenmat33);
```

Comment:
BDBt

Reply:
OK, I will do a new PR

Reply:
It is already done. Take into account that all the comments are Outdated

Reply:
There is a way to hide the Outdated messages